fanboy-addon/fanboy_newsletter_specific_uBO.txt

```diff
@@ -572,6 +572,9 @@ soundproofcow.com##body,html:style(overflow: auto !important; position: initial
 ! monarchmoney.com
 monarchmoney.com##.modal__overlay
 monarchmoney.com##body:style(overflow: auto !important; position: initial !important;)
+! aulart.com
+aulart.com##.elementor-popup-modal
+aulart.com##body:style(overflow: auto !important; position: initial !important;)
```

Comment:
I don't see `position` modified, but `max-height: 100vh;` set for `.dialog-prevent-scroll` class. Personally I prefer using `.dialog-prevent-scroll` but sure FA tends to use `body`.

Reply:
> Should I change this, then?

At least `max-height: 100vh;`